utopia-remix/app/components/projectNotFound.tsx

```diff
@@ -122,7 +144,7 @@ function NotFoundPage({ user, projectId }: { user: UserDetails | null; projectId
               <a
                 href='/projects'
                 rel='noopener noreferrer'
-                style={{ textDecoration: 'none', color: colors.primary }}
+                style={{ textDecoration: 'none', color: isDarkMode ? '#80CAFF' : '#0075F9' }}
```

Comment:
Don't we prefer to use the colors from our palette here? (or maybe add those colors to the palette)